app/views/course/achievements/index.html.slim

```diff
@@ -2,11 +2,8 @@
   = new_button([current_course, :achievement]) if can?(:create, Course::Achievement.new(course: current_course))
 table.table.achievement-list.table-hover
   thead
-    tr
-      th = t('.badge')
-      th = t('.title')
-      th = t('.description')
-      th = t('.requirements')
-      th
+    = render partial: 'achievement_header', locals: { include_requirements: true }
```

Comment:
Don't split the partial like this. Make an achievements partial that does both, the index will just `render partial: 'achievements', local: {...}`